### PR TITLE
Update radiogroup examples: High Contrast Support and APG Coding Practices

### PR DESCRIPTION
**Pull request by jongund**
Updates both radio examples to:
1. Improve color contrast of radio buttons
1. Update coding practices to use APG guidelines
1. Added onclick event for `aria-activedescendant` example
1. Improved high contrast support
1. Reorganized files to use APG example preferred file structure
1. (To-do) Delete obsolete copies of examples in `radio-1` and `radio-2` directories

### Preview Link

[Preview examples in compare branch](https://raw.githack.com/w3c/aria-practices/update-radiogroup-example/examples/radio/radio.html)

### Review checklist

* [x] [Editorial review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#editorial_review) by @mcking65
* [x] [Functional review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#functional_review) by @curtbellew 
* [x] [Visual design review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#design_review) by @a11ydoer 
* [x] [Accessibility review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#accessibility_review) by @charmarkk 
* [x] [Code review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#code_review) by @spectranaut (NOTE: there is a new requirement in the [code review process](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#code-review))
* [x] [Test review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#test_review) by @spectranaut

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/aria-practices/pull/1485.html" title="Last updated on Sep 1, 2020, 7:22 AM UTC (2425d67)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria-practices/1485/145097b...2425d67.html" title="Last updated on Sep 1, 2020, 7:22 AM UTC (2425d67)">Diff</a>

### CODE REVIEW

**Comment by a11ydoer** [comment]
Two comments based on visual review of radio group example

1. Radio group options are **usually laid out vertically, with one choice per line**. However, current example uses a horzontal layout and each option has a clarity for which choice goes with which label. [Referece from Nielsen Norman Group](https://www.nngroup.com/articles/checkboxes-vs-radio-buttons/). 

2. Standard visual representations for "a radio button should be **a small circle that has a solid circle inside it when selected.**" [Referece from Nielsen Norman Group](https://www.nngroup.com/articles/checkboxes-vs-radio-buttons/). Current example has a visual presentaiton of empty circle inside it when selected. The fill can be opposite.

**Comment by jongund** [comment]
@a11ydoer 
@mcking65 
The original radio button example used a vertical layout and solid radio buttons when a button was selected.
But at some point someone in the group and I cannot remember who, changed the layout of the radio buttons to horizontal and  the current bullet design.  The group at that time must have liked the new design better than the traditional radio buttons layout and style, since it was merged.  I would vote for a more traditional looking radio buttons, but maybe Matt remembers more than I do about the change.

**Comment by a11ydoer** [comment]
I agree with @jongund keeping the scope as it is. This is not a deal breaker at all and that is why I used the word, "comment" not "change request"

By the way, this discussion evoked the same quesiton over an over regarding the criteria of "[visual design review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#design_review)" 

"Review the visual presentation to assess whether look and feel is similar to **typical modern implementations** of the widget. Allow for adjustments to accommodate accessibility requirements in cases where **most modern implementations** are not sufficiently accessible."

To the point, what is the criteria for "typycial modern implementations" and "most modern", not that I want to bring in big philosophcal modernism vs post modernism discussion?  :-) 

Should APG examples rather focus on more of widely accepted "a more traditional looking" widget as @jongund mentioned ?

A random thought for Friday afternoon. ;-)

**Comment by mcking65** [comment]
@a11ydoer, If you would like to discuss the meaining of

>Review the visual presentation to assess whether look and feel is similar to typical modern implementations of the widget.

one way would be to raise an issue that highlights questions you have about recently reviewed or changed examples, such as this one. We could then discuss it in a meeting. To make the discussion meaningful, it might be helpful to include screen shots of APG examples that make you think and put them along side screen shots of designs that don't make you think.

**Comment by a11ydoer** [comment]
Thanks @mcking65 
I think it is also a good idea to hear from @ZoeBijl who raised the concern first. Before visual design review process was added, my concern was rather on the fact that APG example pages are not responsive to different screen sizes and we still have the same issue.

**Comment by charmarkk** [comment]
In-progress a11y review:

* JAWS 2020.2006.12
  * Firefox 79 OK
     * Note: Announces first-focused radio button in group twice when tabbed to. (JAWS/Firefox bug or weirdness?) No other buttons receive a double announcement.
   * Chrome 84.0.4147.125 OK
* NVDA 2020.1
     * Firefox 79 OK
     * Chrome 84.0.4147.125 OK

* TO DO
     * Safari/VO
     * Windows HCM

**Comment by jongund** [comment]
@spectranaut 
I have an eslint question/problem related to the use of `use strict` in JS files.
[travis eslint error report](https://travis-ci.com/github/w3c/aria-practices/jobs/3753716960

I am getting the following message for both JS files in this pull request on the line where `"use strict"; is defined:
```
/home/travis/build/w3c/aria-practices/test/tests/radio_radio-activedescendant.js

  3:1  error  Use the global form of 'use strict'  strict

/home/travis/build/w3c/aria-practices/test/tests/radio_radio.js

  3:1  error  Use the global form of 'use strict'  strict

✖ 2 problems (2 errors, 0 warnings)

```
` 

It seems I am doing what all the other JS files for examples are doing with this setting.
Can you help?

**Comment by jongund** [comment]
@charmarkk 
The high contrast issue should be fixed now, can you re-review?

File: test/tests/radio_radio.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-'use strict';
+
```
Comment (spectranaut) [inline test/tests/radio_radio.js:1]: Hey @jongund it looks like use strict was remove here, and that is the cause of the error?

File: test/tests/radio_radio-activedescendant.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-'use strict';
```
Comment (spectranaut) [inline test/tests/radio_radio-activedescendant.js:1]: Same in this file!

**Comment by jongund** [comment]
@carmacleod 
Thanks for the feedback, I updated the documentation.  I would like to change the radio button state from using the esoteric CSS technique to using SVG which will probably fix the Firefox "wobbly" issue.   But Matt does not want that change included in this pull request.

**Pull request merged**